client/src/bastionlab/client.py

```diff
@@ -35,36 +35,57 @@
 
 
 class Client:
-    __bastionlab_torch: "BastionLabTorch" = None
-    __bastionlab_polars: "BastionLabPolars" = None
-    __channel: grpc.Channel
-    __token: bytes
+    """
+    A BastionLab client class that provides access to the BastionLab machine learning platform.
```

Comment:
The class connection is well detailed with its arguments, while this one is not...

Reply:
For new contributors, I would like you to document it.

Reply:
Alright, we will had docs on `Client` too.